telegram/README.md

```diff
@@ -106,6 +106,72 @@ state_name:
     telegram_message: 'New AutoBlind state: [VALUE]'
 ```
 
+#### telegram_info
+
+read (broadcast) a list with specific item-values provided with the attribute.
+The attribute parameter is also the command. 
+e.g. /wetter
+All attribute parameters (commands) are listed with the /info-command in a keyboard menu
+
+#### Simple Example
+
+```
+[Aussentemperatur]
+	name = Aussentemperatur in °C
+	type = num
+	knx_dpt = 9
+	telegram_info = "wetter"
+[Wind_kmh]
+	name = Wingeschwindigkeit in kmh
+	type = num
+	knx_dpt = 9
+	telegram_info = "wetter"
+
+[Raumtemperatur]
+	name = Raumtemperatur Wohnzimmer in °C
+	type = num
+	knx_dpt = 9
+	telegram_info = "rtr_ist"
+
+```
+
+```
+Aussentemperatur:
+    name: Aussentemperatur in °C
+    type: num
+    knx_dpt: 9
+    telegram_info: wetter
+Wind_kmh:
+    name: Wingeschwindigkeit in kmh
+    type: num
+    knx_dpt: 9
+    telegram_info: "wetter"
+
+
+Raumtemperatur
+    name: Raumtemperatur Wohnzimmer in °C
+    type: num
+    knx_dpt: 9
+    telegram_info: rtr_ist
```

Comment:
Are all these quotes required in the configuration (e.g. `telegram_info: "wetter"`)?

Reply:
"name: ... type: ... knx_dpt: ..." are attributes which are defined in the yaml configuration file of the items.
'telegram_info: my_info_name' is a plugin-specific attribute in the item configuration.

Reply:
Yes, but the quotes are usually not necessary.